src/app/components/SelectField.tsx

```diff
@@ -32,38 +32,42 @@ export default function SelectField({
     }
   })
 
-  return (
-    <>
-      <div className={styles.container}>
-        <select
-          ref={selectRef}
-          className={styles.container__input}
-          name="text"
-          value={value}
-          autoComplete="off"
-          title=""
-          required={everFocusedInput}
-          onFocus={() => setEverFocusedInput(true)}
-          onChange={e => {
-            if (e.target.selectedIndex !== -1) setІsChoosen(true)
-            if (setValue) setValue(e.target.value)
-          }}
-          style={{ content: '↓', color: 'white' }}
-        >
-          <option value="" disabled></option>
-          {list.map((list: { name: string }) => {
-            const { name } = list
+  const handleOnFocus = () => setEverFocusedInput(true)
+
+  const handleOnChange = (e: {
+    target: {
+      selectedIndex: number
+      value: React.SetStateAction<string | undefined>
+    }
+  }) => {
+    if (e.target.selectedIndex !== -1) setІsChoosen(true)
+    if (setValue) setValue(e.target.value)
+  }
 
-            return (
-              <option key={name} value={name}>
-                {name}
-              </option>
-            )
-          })}
-        </select>
-        <label className={labelStyle}>{placeholder}</label>
-      </div>
-    </>
+  return (
+    <div className={styles.container}>
+      <select
+        ref={selectRef}
+        className={styles.container__input}
+        name="text"
+        value={value}
+        autoComplete="off"
+        title=""
```

Comment:
Why we need to explicitly say that `title` is empty string. Can we just remove that line?

Reply:
Yes, we can.